### PR TITLE
TF-6 Refactoring: remove ts-loader

### PR DESCRIPTION
**Pull request by lyykfi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
 	"scripts": {
-		"build-master": "node ./node_modules/webpack/bin/webpack.js",
-		"build-staging": "node ./node_modules/webpack/bin/webpack.js --mode none",
-		"start": "node ./node_modules/webpack/bin/webpack.js --watch",
-		"lint": "node ./node_modules/eslint/bin/eslint.js --ext .ts ./src",
+		"build-master": "npx webpack --mode=production",
```
Comment (edwardupton) [inline package.json:3]: Is `mode=production` the same as `mode production`?

File: package.json [hunk 2]
```diff
@@ -36,10 +37,9 @@
 		"prettier": "1.18.2",
 		"sinon": "2.4.1",
 		"sinon-chai": "2.12.0",
-		"terser-webpack-plugin": "5.1.1",
-		"ts-loader": "6.2.2",
+		"terser-webpack-plugin": "^5.1.1",
 		"ts-node": "9.1.1",
-		"typescript": "3.9.9",
+		"typescript": "^4.2.4",
```
Comment (edwardupton) [inline package.json:42]: Does eslint typescript plugin need updating too?

**Review by edwardupton — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lyykfi) [reply package.json:3]: Should be the same.

File: package.json (hunk 2 above)
Comment (lyykfi) [reply package.json:42]: Good point!

**Pull request merged**